templates/commands/render/action.go

```diff
@@ -167,6 +167,34 @@ func processGlobs(ctx context.Context, paths []model.String, fromDir string) ([]
 	return out, nil
 }
 
+// TODO (#158): There could be some refactor we can do to combine processGlobs
```

Comment:
I am deferring the refactor in later PRs since we might want to handle "glob <glob> did not match any files" error differently, as in ignore, this error should be ignored.

Reply:
I think it's a pretty simple refactor, and perhaps simple enough to implement now, rather than copy-pasting this function with a small change.

The return type of `processGlobs` could change from `([]model.String, error)` to `([]model.String, bool, error)`. The boolean would be `true` if there were results, or `false` if not. Each caller of this function would decide whether `false` is an error.

In general, I think copy-pasting code with minor changes creates lots of problems and should be strenuously avoided.

Reply:
I thought about that approach, I was thinking to return empty list instead. I hesitated because:
1. we won't be able to report the information of the position of the pattern unless we return more information.
2. we have a default ignore list that is []string and does not have position information, unless we convert []string to []model.String with empty positions but it doesn't make sense to me.

based on 2, I think processGlobsString is needed, what do you think?

Reply:
Removed as not needed anymore.